mappings/net/minecraft/client/util/math/MatrixStack.mapping

```diff
@@ -1,26 +1,54 @@
 CLASS net/minecraft/unmapped/C_cnszsxvd net/minecraft/client/util/math/MatrixStack
+	COMMENT Represents a stack of matrices.
+	COMMENT <p>
+	COMMENT You can {@link MatrixStack#push() push} and {@link MatrixStack#pop() pop} matrices onto the stack.
+	COMMENT <p>
+	COMMENT All other operations are applied to the last matrix pushed onto the stack.
+	COMMENT Those operations can be: translation, scaling, rotation, multiplication with other matrices, etc.
```

Comment:
Grammar nitpick, but I believe `scaling` should just be `scale` here to be consistent with the noun forms of the other operations. 

Alternatively, changing them all to verb form is slightly fewer characters: `translate, scale, rotate`... (scale is noun *and* verb form)

Reply:
Should be fixed!